test/src/code_field/code_controller_tab_test.dart

```diff
@@ -20,6 +22,26 @@ public class MyClass {
    public void main() {
    }
 }
+''';
+
+  const namedSectionsWithTabs = '''
+class MyClass {
+\tvoid readOnlyMethod() {// [START section1]
+\t}// [END section1]
+\t// [START section2]
+\tvoid method() {
+\t}// [END section2]
+}
+''';
+
+  const namedSectionsWithDoubleSpaces = '''
+class MyClass {
+  void readOnlyMethod() {// [START section1]
+  }// [END section1]
+  // [START section2]
+  void method() {
+  }// [END section2]
+}
 ''';
 
   bool _areIdenticalTexts(String text1, String text2) {
```

Comment:
Can we get rid of this and just use `expect(text1, text2)`?

Reply:
Im most cases except one :)

Reply:
Yes, deleted